src/baguetteBox.js

```diff
@@ -149,8 +149,14 @@
         [].forEach.call(gallery, function(galleryElement) {
             if(userOptions && userOptions.filter)
                 regex = userOptions.filter;
+
+            if(galleryElement.tagName === 'A')
```

Comment:
This needs formatting, brackets, spaces, etc.

Also, you shouldn't declare the var like that; it can be misleading. Better declare it outside and set its value then.